crates/sui-indexer-alt-restorer/src/lib.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+mod archives;
+mod snapshot;
+
+use archives::ArchivalCheckpointInfo;
+use clap::Parser;
+use sui_pg_db::DbArgs;
+
+use crate::snapshot::SnapshotRestorer;
+
+#[derive(Parser, Debug, Clone)]
+#[clap(name = "sui-indexer-restorer")]
+pub struct Args {
+    /// Restore from end of this epoch.
+    #[clap(long, env = "START_EPOCH", required = true)]
+    pub start_epoch: u64,
+
+    /// Endpoint to fetch snapshot files from.
```

Comment:
What kind of string is the endpoint? Is it a URL, or an enum (like `GCS` vs `S3` etc?) Clarifying this in the doc comment with some examples would be helpful.